lib/ui/widget/member_tile.dart

```diff
@@ -67,11 +67,19 @@ class MemberTile extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final style = Theme.of(context).style;
-
+    final bool isOnline = _me
+        ? myUser?.online ?? false
+        : user?.user.value.online ?? false;
     return ContactTile(
       user: user,
       myUser: myUser,
       dense: true,
+      subtitle: [
+        Text(
+          isOnline ? 'label_online'.l10n : 'label_offline'.l10n,
+          style: style.fonts.normal.regular.secondary,
+        ),
+      ],
```

Comment:
1) This makes every single `MemberTile` to have the `label_online` or `label_offline` labels. Those aren't used only on `ChatInfo` page. Is it intended? It might be actually, just be sure to do this decision intentionally. AFAIK it is used in `Call` as well - there it might fit well as well.
2) When offline, user has the date they were last online. Shouldn't it be displayed instead of `offline`?